cmake/public/cuda.cmake

```diff
@@ -194,7 +194,7 @@ add_library(torch::cudart INTERFACE IMPORTED)
 if(CAFFE2_STATIC_LINK_CUDA)
     set_property(
         TARGET torch::cudart PROPERTY INTERFACE_LINK_LIBRARIES
-        "${CUDA_TOOLKIT_ROOT_DIR}/lib64/libcudart_static.a" rt dl)
+        "${CUDA_TOOLKIT_ROOT_DIR}/lib64/libcudart_static${CMAKE_STATIC_LIBRARY_SUFFIX}" rt dl)
```

Comment:
Change this to "${CUDA_TOOLKIT_ROOT_DIR}/${CUDA_LIB_PATH}/${CMAKE_STATIC_LIBRARY_PREFIX}cudart_static${CMAKE_STATIC_LIBRARY_SUFFIX}"?

Reply:
I noticed that `find_package(CUDA)` gives a variable `CUDA_cudart_static_LIBRARY` that might clean this up a bit. However, it occurs to me that windows probably doesn't have `rt` or `dl` either. Any idea if it can just be left off windows?

Reply:
that's fine, I think.